src/main/groovy/wooga/gradle/unity/batchMode/BaseBatchModeSpec.groovy

```diff
@@ -17,9 +17,7 @@
 
 package wooga.gradle.unity.batchMode
 
-import org.gradle.api.internal.IConventionAware
-
-interface BaseBatchModeSpec extends IConventionAware {
```

Comment:
No longer needed? I thought we just added this here.

Reply:
will roll it back!